clientApp/src/app/components/edit-user-data/edit-user-data.component.html

```diff
@@ -0,0 +1,47 @@
+<div fxLayout="row"
```

Comment:
this side is only available when you know the url or the User Data Guard will redirected you to this side and what if during your lifetime write a PhD thesis or change your surname? i think there suposuped to by an option to reatch the side without knowing the url